src/bencode/core.clj

```diff
@@ -314,14 +314,15 @@
   'namespace/name'."
   (fn [_output thing]
     (cond
-      (instance? (RT/classForName "[B") thing) :bytes
+      (nil? thing) :list
+      (-> thing class .getComponentType (= Byte/TYPE)) :bytes
```

Comment:
Might be a good idea to add a comment here we've borrowed this from Clojure 1.9.

Reply:
Done.